roles/crash_files/tasks/crash_files.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Execute crash log check on Cisco IOS XR device (logging)
```

Comment:
lets try to have a common task where we can include or call specific commands based on OS.